src/data/showcases.js

```diff
@@ -233,6 +233,15 @@ export const Tags = {
 // Please don't add the "favorite"-tag yourself.
 // Provide pure NFT projects only get the tag NFT, not "tokens"
 export const Showcases = [
+  {
+    title: "Reverse DJED",
+    description:
+        "Mint and burn DJED, Cardano's overcollateralized stablecoin, with our open-source platform. Transparent alternative to DJED.xyz - accessible 24/7 anywhere. Built by Artifex Labs for the Cardano community.",
```

Comment:
1. Unless it's easily proven _and_ common knowledge that DJED is overcollateralised, the description shouldn't mention that term.
2. `accessible 24/7 anywhere` is irrelevant (unless you can indicate otherwise) since I can't see any country or time restrictions on `DJED.xyz`.
3. The promotional phrase `Built by Artifex Labs for the Cardano community` should be dropped since a) generally other Showcase items don't plug the developer in this way, and b) all these offerings are "for the Cardano community" so this statement is redundant at the least.

Reply:
@kylixafonso thanks for putting your statements about the conventional DJED into context.

Since many of the original Dev Portal listings were lifted directly from promotional language on other web sites & GitHub repos, I couldn't fairly insist that you change it... though I believe all the examples you mention would ideally be cleaned up in a "writing style" overhaul which I don't think anyone feels they have the authority to do.